codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/SourceGenerator.scala

```diff
@@ -253,20 +253,10 @@ object SourceGenerator extends PrettyPrinter {
     }
 
     val serviceImports = model.services.values.flatMap { serv =>
-      val viewWithTransforms =
-        serv match {
-          case view: ModelBuilder.ViewService => view.transformedUpdates.nonEmpty
-          case _ => false
-        }
-      if (serv.fqn.parent.javaPackage != mainClassPackageName) {
-        val outerClass = s"${serv.fqn.parent.javaPackage}.${serv.fqn.parent.javaOuterClassname}"
-        if (viewWithTransforms)
-          List(serv.fqn.fullName, outerClass)
-        else
-          List(outerClass)
-      } else List.empty
+      if (serv.fqn.parent.javaPackage != mainClassPackageName)
+        serv.fqn.fullName :: s"${serv.fqn.parent.javaPackage}.${serv.fqn.parent.javaOuterClassname}" :: Nil
+      else List.empty
```

Comment:
This changed in #218, but now reverted to what we have in `main`. Not clear why we added a flag  (`viewWithTransforms`). 

In any case, we do need to include `serv.fqn.fullName` in the imports for the Actions.

Reply:
Got to revert this. Tests were failing. 🤷